httpx/middleware/query_normalizer_test.go

```diff
@@ -0,0 +1,141 @@
+package middleware_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"net/url"
+
+	"github.com/msales/pkg/httpx/middleware"
+	"github.com/stretchr/testify/assert"
+)
+
+var testsData = []struct {
```

Comment:
I wont fail for this, but it is unclear which test this belongs to. Consider putting it in the test function.

Reply:
There is only one test.